afqinsight/nn/performance_utils.py

```diff
@@ -0,0 +1,328 @@
+import tempfile
```

Comment:
Make all these part of the existing "utils.py" module. We'll use them as general utility functions